### PR TITLE
Fix minor UG issues

### LINKED ISSUE #143: [PE-D][Tester C] Formatting error / typo in 'view' section of UG
I think the dash is a typo...?

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: nananakx-x/ped#14

### PR DESCRIPTION
**Pull request by nabonitasen**
Closes #143, #148, #125.

This PR updates minor issues in the UG.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ You can click on any of the links below to navigate to the respective sections f
     - [7.2. Command Summary](#7-2-command-summary)
       - [7.2.1. Basic applicant management commands](#7-2-1-basic-applicant-management-commands)
       - [7.2.2. Applicant comparison and evaluation commands](#7-2-2-applicant-comparison-and-evaluation-commands)
-      - [7.2.3. Data management and export commands](#7-2-3-data-management-and-export-commands)
+      - [7.2.3. Data management and import commands](#7-2-3-data-management-and-import-commands)
```
Comment (amosting) [inline docs/UserGuide.md:64]: I think can just say data management commands to simplify things HAHA

File: docs/UserGuide.md [hunk 2]
```diff
@@ -1268,8 +1268,8 @@ If your changes to the data file makes its format invalid, TAfinder will discard
 | **Comment**             | - `comment INDEX COMMENT`<br/> - e.g., `comment 3 Hardworking`                    |
 
 
-#### 7.2.3. Data management and export commands
+#### 7.2.3. Data management and import commands
 | Action                  | Format, Examples                                                |
 |-------------------------|-----------------------------------------------------------------|
-| **Export**              | - `export FILENAME`<br/> - e.g., `export ta-applicants.csv`     |
+| **Import**              | - `import FILENAME`<br/> - e.g., `import ta-applicants.csv`     |
```
Comment (amosting) [inline docs/UserGuide.md:1274]: Wait are we removing the export feature?

**Review by amosting — APPROVED** [review]
Small nits to fix if not LGTMMMMMM!!!!

File: docs/UserGuide.md (hunk 2 above)
Comment (nabonitasen) [reply docs/UserGuide.md:1274]: we don't actually have an export feature, we did import instead

**Review by ravern — APPROVED** [review]
LGTM

**Pull request merged**